lib/check-self-update/fetch-module-registry-info.js

```diff
@@ -0,0 +1,16 @@
+const parseJson = require('./parse-json');
+const requestPromise = require('./request-promise');
+const getModuleRegistryUrl = require('./get-module-registry-url');
+
+/* Fetches a json object with module info when given its name */
+
+const fetchModuleRegistryInfo = (name) => {
+  return new Promise((resolve, reject) => {
```

Comment:
@rafaelcalpena The promise wrapper is unnecessary since your `requestPromise` function already returns a promise.